src/luminol/algorithms/correlator_algorithms/cross_correlator.py

```diff
@@ -87,7 +88,7 @@ def _find_allowed_shift(self, timestamps):
         param list timestamps: timestamps of a time series.
         """
         init_ts = timestamps[0]
-        residual_timestamps = map(lambda ts: ts - init_ts, timestamps)
+        residual_timestamps = list(map(lambda ts: ts - init_ts, timestamps))
```

Comment:
A list comprehension would probably be more readable: `[ts - init_ts for ts in timestamps]`.

Reply:
Agreed.